xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3627,7 +3627,7 @@ platform.linkchecker.livetable.date=Last Checked
 
 ### Dashboard Application Resources
 platform.dashboard.user.preferences=Dashboard preferences
-platform.dashboard.user.displayOnMainPage=Display my dashboard on the wiki home when I'm logged in (instead of the default dashboard)
+platform.dashboard.user.displayOnMainPage=Display my dashboard when I'm logged in (instead of the default dashboard)
```

Comment:
I think "Replace the default dashboard with my custom dashboard" sound better.
WDYT?